test/jdk/java/awt/TrayIcon/DisposeInActionEventTest.java

```diff
@@ -57,24 +57,26 @@ public static void main(String[] args) throws Exception {
         String clickInstruction =
                 (Platform.isOSX()) ? "Right-click" : "Double click (left mouse button)";
```

Comment:
On Linux, it triggers with a single click(Tested on Ubuntu 23.10)

Reply:
Manual tests that require a tester's interpretation and that are specific to the way a platform expects user interaction can be  subjective but the idea is that they are objective. I already didn't like the clickInstruction logic, now I  like it even less.